sorc/ncep_post.fd/grib2_module.f

```diff
@@ -268,7 +276,8 @@ subroutine gribit2(post_fname)
 !
 !--- sequatial write if the number of fields to write is small
 !
-    if(minval(nfld_pe)<1.or.num_procs==1) then
+!JESSE    if(minval(nfld_pe)<1.or.num_procs==1) then
+    if(num_procs==1) then
```

Comment:
Given this change you made, I think you should comment out lines 284, and 286-291, right?